### PR TITLE
[bitnami/kong] readme: add dbless mode infos

### LINKED ISSUE #5778: [bitnami/kong] Document how-to enable db-less mode (readme)
**Which chart**:
kong: 3.4.4

**Is your feature request related to a problem? Please describe.**
There isn't info about how to enable Kong in db-less mode as described in the official guide.

**Describe the solution you'd like**
I write here the snippet to add on the README.md of the project. The solution is tested (and it works) by me.
Anything else is necessary in order to enable db-less kong mode.

```
### DB-less

#### TL; DR
1. Set `Value.database` with any value other than "postgresql" or "cassandra". For example `Value.database: "off"`
2. Set
```
Value.kong.extraEnvVars:
- name: KONG_DATABASE
  value: "off"
```

#### Short Story
Kong 1.1 added the capability to run Kong without a database, using only in-memory storage for entities: we call this DB-less mode. When running Kong DB-less, the configuration of entities is done in a second configuration file, in YAML or JSON, using declarative configuration (ref. [Link](https://docs.konghq.com/gateway-oss/1.1.x/db-less-and-declarative-config/)).
On the step 4 of [kong official docker installation](https://docs.konghq.com/install/docker#db-less-mode),
just change the config of docker run command FROM -e "KONG_DATABASE=postgres" TO -e "KONG_DATABASE=off" and you can go ahead!
```

I've also attached the full readme (updated with the new chapter).

[README.txt](https://github.com/bitnami/charts/files/6136958/README.txt)

### PR DESCRIPTION
**Pull request by texano00**
**Description of the change**
There isn't info about how to enable Kong in db-less mode as described in the official guide.
I write here the snippet to add on the README.md of the project. The solution is tested (and it works) by me.
Anything else is necessary in order to enable db-less kong mode.

**Benefits**
Enable dbless mode.

**Possible drawbacks**
None

**Applicable issues**

  - fixes #5778

**Additional information**
None

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/kong/README.md [hunk 1]
```diff
@@ -310,6 +310,21 @@ The Bitnami Kong chart allows setting two database backends: PostgreSQL or Cassa
...
+```
+Value.kong.extraEnvVars:
+- name: KONG_DATABASE
+  value: "off"
+```
+
+#### Short Story
+Kong 1.1 added the capability to run Kong without a database, using only in-memory storage for entities: we call this DB-less mode. When running Kong DB-less, the configuration of entities is done in a second configuration file, in YAML or JSON, using declarative configuration (ref. [Link](https://docs.konghq.com/gateway-oss/1.1.x/db-less-and-declarative-config/)).
+As is said in the step 4 of [kong official docker installation](https://docs.konghq.com/install/docker#db-less-mode),
+just add the env variable "KONG_DATABASE=off".
```
Comment (carrodher) [inline bitnami/kong/README.md:327]:
```suggestion

### DB-less

Kong 1.1 added the capability to run Kong without a database, using only in-memory storage for entities: we call this DB-less mode. When running Kong DB-less, the configuration of entities is done in a second configuration file, in YAML or JSON, using declarative configuration (ref. [Link](https://docs.konghq.com/gateway-oss/1.1.x/db-less-and-declarative-config/)).
As is said in step 4 of [kong official docker installation](https://docs.konghq.com/install/docker#db-less-mode), just add the env variable "KONG_DATABASE=off".

#### How to enable it

1. Set `database` value with any value other than "postgresql" or "cassandra". For example `database: "off"`
2. Use `kong.extraEnvVars` value to set the `KONG_DATABASE` environment variable:
```yaml
kong.extraEnvVars:
- name: KONG_DATABASE
  value: "off"
```
```

**Review by carrodher — COMMENTED** [review]
Thanks for your contribution! I just suggested some small changes to follow the same structure as other sections. Can you take a look?

**Comment by texano00** [comment]
OK @carrodher

**Comment by carrodher** [comment]
You can directly apply the suggestion or we can do it on our side, as you prefer. Thanks again!

**Comment by texano00** [comment]
@carrodher it's ready

**Comment by carrodher** [comment]
I'm sorry but it seems you marked as resolved the conversation but the changes were not applied to the file. You need to click on "Commit suggestion" or apply the changes in a manual way from your branch

**Pull request merged**